lerna-debug.log

```diff
@@ -0,0 +1,52 @@
+lerna(verbose) GitUtilities.isInitialized    ()
```

Comment:
Obviously it's too early to make an actual code review, lot's of stuff are not clean :)
Thanks for reporting.